### PR TITLE
feat: use otto for js templating

### PR DESCRIPTION
**Pull request by adityathebe**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -12,12 +12,13 @@ require (
...
 	github.com/itchyny/gojq v0.12.13
 	github.com/pkg/errors v0.9.1
+	github.com/robertkrimen/otto v0.2.1
 	github.com/stretchr/testify v1.8.4
 	github.com/ugorji/go/codec v1.2.11
 	golang.org/x/text v0.11.0
 	golang.org/x/tools v0.7.0
 	gotest.tools/v3 v3.4.0
-	k8s.io/apimachinery v0.27.3
+	k8s.io/apimachinery v0.26.4
```
Comment (moshloop) [inline go.mod:21]: Don't bump these

Comment (adityathebe) [reply go.mod]: @moshloop All other repos use `v0.26.4`. Else I'd have to bump it up on canary checker

Comment (moshloop) [reply go.mod]: @adityathebe  ok - then downgrade k8s.io/api as well

**Comment by moshloop** [comment]
@adityathebe can we add tests ?

**Pull request merged**